pkg/sypgp/sypgp.go

```diff
@@ -349,6 +350,68 @@ func StorePubKey(e *openpgp.Entity) (err error) {
 	return
 }
 
+// compareLocalPubKey : compares a key ID with a string. return true if the
+// key / oldToken *dont* match.
+func compareLocalPubKey(e *openpgp.Entity, oldToken string) bool {
+	if fmt.Sprintf("%X", e.PrimaryKey.Fingerprint) != oldToken {
```

Comment:
note that you are evaluating a boolean to return a boolean. In other words, you have this structure:

    if condition {
        return true
    }
    return false

that's equivalent to:

    return cond